lib/check_search.rb

```diff
@@ -60,6 +56,18 @@ def initialize(options, file = nil, team_id = Team.current&.id)
     'fact_check_published_on' => 'fact_check_published_on'
   }
 
+  def adjust_keyword_filter
+    unless @options['keyword'].blank?
+      # This regex removes all characters except letters, numbers, and whitespace in any language - stripping out special characters can improve match results
+      @options['keyword'].gsub!(/[^[:word:]\s]/, ' ')
```

Comment:
@melsawy does it work for Arabic?

Reply:
yes and confirmed locally with Arabic text that contains special character.